### PR TITLE
Simplify the UI class.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/ui.toit [hunk 1]
```diff
@@ -49,68 +49,83 @@ create-ui-from-args_ args/List:
...
+  Emits the given $map of the given message-$kind.
+
+  This method is only called if the printer does not request a structured
+    representation.
+  */
+  emit-map --kind/int map/Map --title/string?
+
+  /** Whether the printer wants a structured representation for the given $kind. */
+  needs-structured --kind/int -> bool
+  /** Emits the given $json-object of the given message-$kind. */
```
Comment (kasperl) [inline src/ui.toit:83]: Add newline here for consistency?

File: src/ui.toit [hunk 2]
```diff
@@ -49,68 +49,83 @@ create-ui-from-args_ args/List:
...
+  emit-structured --kind/int json-object/any
+
```
Comment (kasperl) [inline src/ui.toit:85]: Drop one newline.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/ui.toit (hunk 1 above)
Comment (floitsch) [reply src/ui.toit:84]: done.

File: src/ui.toit (hunk 2 above)
Comment (floitsch) [reply src/ui.toit:86]: done.

**Pull request merged**